data/os/RedHat/VirtuozzoLinux.yaml

```diff
@@ -0,0 +1,219 @@
+# VirtuozzoLinux default repos for AWS.
+---
+yum::os_default_repos:
+  - "rhui-REGION-rhel-server-releases"
```

Comment:
Can you have a look at the CentOS.yaml? I think the default repositories here are not correct for VirtuozzoLinux.

Reply:
I re-checked `init.pp`. To my understanding, these files would only be necessary if someone uses `manage_os_default_repos = true`. This would not work with the current version of this module. So I'd expect noone is actually using this.

If this is correct, implementing correct `VirtuozzoLinux.yaml`. `VirtuozzoLinux/6.yaml`, and `VirtuozzoLinux/7.yaml` would not be worth the time.

(I tried it. 6 and 7 only share two repo file names, everything else is different)